### ISSUE TITLE
Icon eye marks buttons focus and active style

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#571]
The new "icon eye" buttons to set or remove the marks in the editor need a focus style for accessibility. I'd propose to use the same "circular focus" style used for the Help buttons. Something like this:

<img width="860" alt="screen shot 2016-05-31 at 16 57 38" src="https://cloud.githubusercontent.com/assets/1682452/15679861/ba60dd90-2753-11e6-9624-2a5be16453c0.png">

This would require to make the clickable area a bit bigger (which is always a good thing) and then the positioning of the buttons should be adjusted a bit I guess.

**Comment by afercia** [comment]
Chatted a bit about this, agree to try to have a better "active" style too. The idea would be to make the buttons clickable area a bit bigger and rounded and use the Yoast purple. A very rough mockup:

<img width="633" alt="screen shot 2016-06-01 at 11 33 28" src="https://cloud.githubusercontent.com/assets/1682452/15707724/8d534fc0-27fa-11e6-9a22-4d9150f5f891.png">
